src/core/smallvec_type.hpp

```diff
@@ -158,6 +158,23 @@ class SmallVector {
 		}
 	}
 
+	/**
+	 * Insert a new item at a specific position into the vector, moving all following items.
+	 * @param item Position at which the new item should be inserted
+	 * @return pointer to the new item
+	 */
+	inline T *Insert(T *item)
```

Comment:
You introduce this function, but you don't use it; was that intended?

Reply:
Yes and no :) This is some old backlog cleanup, and Insert() is used in some other commits.